dev-scripts/ocsetup

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+getversion() {
+        if [ ! -f $1/etc/org.apache.karaf.features.cfg ]; then
+          exit 1
+        fi
+        sed -n 's/.*opencast-karaf-features\/\([0-9]*\).*/\1/p' $1/etc/org.apache.karaf.features.cfg
+}
+
+setup() {
+        version=$(getversion $1)
+        if [ -z "$version" ]; then
+          echo "$1/etc/org.apache.karaf.features.cfg is not a path, unable to process"
+          return 1
+        fi
+        echo "Processing $1, detected OC $version"
+        if [ "$version" -ge "9" ]; then
+                docker compose -f $COMPOSEFILE up -d
+        fi
+        docker compose -f ../docker-nginx-cors/docker-compose.yml up -d
```

Comment:
I just remembered that this depends on having a local docker proxy going from `localhost:8080` to `localhost`.  Personally I'm using git@github.com:maximillianfx/docker-nginx-cors.git with a patch set to set things up for Opencast.  Not sure how to publish that.  Reviewer, any thoughts?